### DISCUSSION TITLE
UG DQ 01 — Ontology and Intelligence Analysis

### DISCUSSION THREAD

**Discussion by johnbeve** [Applied-Ontology-Education/Ontology-and-Intel-Analysis-Fall-2024#1] [Undergraduate Discussion Questions]
**GitHub Discussion title:** `UG DQ 01 — Ontology and Intelligence Analysis`

**Maximum length:** 500 words

Complete the Week 1 readings and lectures before responding.

## Prompt

Identify one problem in intelligence analysis for which **ambiguity, inconsistent terminology, or unclear categorization** could plausibly affect analysis.

Answer the following:

1. What is the intelligence-analysis problem?
2. What terms or categories are likely to be used inconsistently or ambiguously?
3. What would an ontology contribute that a simple list of terms or database schema would not?
4. What is one question an analyst should be able to answer if the domain were modeled successfully?

Use at least one idea from the Week 1 readings or lectures.

**Comment by adriankoz-jos** [comment]
A good example of a way in which unclear categorization can plausibly affect analysis is in determining the number of potential insurgents or partisans in a given populace. Assumptions or inherent biases around what resistance looks like and who is capable of resistance can result in overly broad definitions that risk the authorization of force against innocents: for example, the standard American practice of defining any males between the ages of 15 and 70 as "military age" reclassifies them as valid targets in lieu of exonerating evidence. Alternatively, overly restrictive definitions that can potentially hamstring counter-insurgent efforts, as with Nazi soldiers in Vichy France, who susceptible to being seduced and either murdered or captured by female members of the resistance.
These definitional discrepancies, especially as applied to mid and low-level ontologies, can result in blind spots. In the absence of further evidence provided by a well-modeled ontology, the younger brother and grandfather of a suspected insurgent could both be deemed valid targets by association, regardless of their level of involvement in partisan activities. Likewise, the insurgent's actively involved girlfriend or sister may be overlooked by an occupying force using an unnecessarily narrow definition of resistance. 
A list of terms or database schema are less than ideal for mapping these sort of relations compared to an ontology: the list of terms is completely unable to model the relationship between actors, and a database schema may have an inconsistent naming convention or be unsuited to modeling the relations between actors in a way that mutual connections can be analyzed reliably. 
A well-modeled ontology should be able to show with significantly more certainty the ways in which any given individual is related to a network of known insurgents, dramatically reducing the likelihood of an innocent being targeted. An analyst using this ontology should be able to ascertain how many points of contact the aforementioned insurgent's 15 year old brother has with known partisan operators: it may be the case that the insurgent's activities are withheld from the younger members of his family, or that his girlfriend was his introduction to the insurgent network, and the rest of his family are unaware. This level of context is unbelievably beneficial when making decisions that can result in the apprehension or even death of suspected bad actors.

**Comment by LaibahA** [comment]
One situation related to intelligence analysis might be figuring out who among a big team is a spy or mole. Inconsistent or ambiguous terminology can cause reliability issues, for example if different sources use different category labels to describe how suspicious or not suspicious an individual is. For example, two sources might label an individual they believe to be innocent and in the lowest possible suspicion category differently, one saying they are “less suspicious” and the other saying they are “not suspicious”. This can be tricky to interpret and an analyst might falsely believe that one source has some suspicion about the individual and another believes they are not suspicious at all, when in reality, both sources believe the individual is in the clear. 

An ontology can address the ambiguity and inconsistency between the datasets by providing a standardized set of meanings and relationships to the data. This is related to the concept, Semantic Enhancement that was discussed in the readings. Different databases could be using various labels for concepts such as individuals with clear alibis, people involved in the situation, people who are suspicious, and people who are most likely to be the mole/spy, leading to inconsistent labelling of the varying range of suspicion around individuals. Additionally, a simple list of terms couldn’t represent the levels of hierarchy or connections between the data. Without shared meaning of the label terminology, analysts have to integrate lots of data that may seem inconsistent and hard to interpret clearly, and can miss out on important meaning and relationships present in the data. But, with SE, the data can be interpreted according to labels applied to the data that have standardized meaning. Additionally, ontology represents the relationships between the concepts present in the data, which would further help an analyst reason about different levels of suspicion present in the data. Note that the data itself isn’t changing, SE is just providing semantic annotations to help with interpretation. 

If the domain were modeled successfully, an analyst could ask questions like “Which individuals are not considered suspicious at all across all datasets?”, or, “Which individuals are considered most likely to be the spy/mole based on the evidence across all datasets?”. Note that it’s not guaranteed that every individual is in all the datasets, so the analyst would need to use reasoning and also look at the relationships or available information in the datasets to come to a conclusion like that. Since there’s a common layer of meaning given to all the data through the labels based on the terminology provided from the ontology, we are able to connect, search, and reason over all the isolated datasets as if they actually did share a common vocabulary. This means that an analyst doesn’t have to understand the varying terminology used by each source.

**Comment by drewbonge12** [comment]
A way in which ambiguity, inconsistent terminology, or unclear categorization will affect analysis could be seen in identifying the likelihood of a potential threat through estimative words such as “likely”, “less likely”, “few”, and “frequent”. With improper communication between agencies, an analyst may come across occasions where estimative words are valued differently; Such as, “likely” being valued at >50% probability by one agency, while being valued at >80% by another agency. It now becomes the job of the analyst to make sense of the variation in data supplied by multiple agencies because of inconsistency in terminology. In a scenario where you have personnel deployed and there is a “likely” attack, the variation as seen above could mean the difference in preparation to ward off an ambush. Specifically, when dealing with human personnel, a small variation in probability could result in unnecessary injury or death.
With use of an ontology, it is possible to disambiguate values that are assigned to estimative words. Making use of mid-level ontologies would allow for agencies or departments to utilize a standardized form of terms, as well as relationships between other concepts. A mid-level ontology would create a joint set that agencies or departments could use interoperably to refer to terms such as “likely” in harmony whereas without one, there is room for variation in definitions throughout individually engineered databases. With individual data without an ontology, two different agencies could both be using the term “likely”, while meaning two entirely different things. This will pose the problem of ambiguity upon intelligence analysts, tasking them to account for the discrepancies in data they are receiving. In this case, intelligence will potentially be unsuccessful and could result in too much preparation, or too little preparation to defend from an incoming attack.
Making use of a successful ontology, an analyst will be able to answer the question of how estimative words such as "likely" should be assigned a probability will become more consistent. If all agencies that are relaying information to an analyst use an ontology with a defined probability lexicon, they dramatically reduce the risk of inconsistency. With this ontology, an analyst can be certain the term “likely” will consistently be assigned the same probability value when seen in intelligence given from agency to agency. There will be no need to contemplate the discrepancies seen throughout agencies, thus resulting in a more precise and efficient process. This will in turn ensure safety when dealing with the probability of ambush on human personnel.
